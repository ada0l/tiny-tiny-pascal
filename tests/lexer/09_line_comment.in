// Hello, world
x := 10;
x += 10; // add 10
// Hello, world // x -= 20;