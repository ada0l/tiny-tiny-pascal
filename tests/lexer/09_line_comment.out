2	1	Id	x	x
2	3	Operation	ASSIGN	:=
2	6	Integer	10	10
2	8	Separator	SEMICOLON	;
3	1	Id	x	x
3	3	Operation	ADDASSIGN	+=
3	6	Integer	10	10
3	8	Separator	SEMICOLON	;
4	29	EOF	EOF	EOF